docs/version0.33.md

```diff
@@ -33,6 +33,7 @@ The following new features and notable changes since version 0.32.0 are included
 - ![Start of content that applies to Java 17 plus](cr/java17plus.png) [Default operating system stack size increased on x64 platforms](#default-operating-system-stack-size-increased-on-x64-platforms)
 - [Control groups v2 support](#control-groups-v2-support)
 - [Support for OpenSSL 3.0.x](#support-for-openssl-30x)
+- [Support for EC key agreement algorithm by using native OpenSSL library added](#support-for-ec-key-agreement-algorithm-by-using-native-openssl-library-added)
```

Comment:
Needs "the". I suggest you use the same format as has been used in the past, see eg https://eclipse.dev/openj9/docs/version0.15/.